ietf/templates/includes/header_iab.html

```diff
@@ -10,18 +10,18 @@
     >
         <div class="container container--iab">
             <!-- Brand Image -->
-            <div class="position-relative d-flex align-items-center">
-                <div class="navbar-brand">
-                    <a class="text-light stretched-link" href="{{ HOME.url }}">
-                        <img
-                            class="header__logo"
-                            src="{% static 'img/iab-logo.svg' %}"
-                            alt="IAB logo"
-                        />
-                    </a>
-                </div>
-                <!-- Brand Text -->
-                <div class="iab-logo-text">Internet Architecture Board</div>
+            <div class="navbar-brand">
```

Comment:
This looks to me like the one spot where wrong code (apart from redundant css classes) was merged in the branch. These changes make the logo text clickable as well as the logo image, so if that no longer happens on your branch you'll want to reinstate the deleted code.